src/InstallationValidator.Core/Services/BatchOutputLoader.cs

```diff
@@ -72,6 +72,7 @@ private BatchSimulationExport mapToBatchSimulationExport(OldBatchSimulationExpor
                Values = outputValues.Values,
                Dimension = outputValues.Dimension,
                Unit = dimension.DefaultUnitName,
+               ComparisonThreshold = outputValues.ComparisonThreshold
```

Comment:
@Yuri05 Here was the culprit. I assumed that v12 at least would not be a corrupted file. Let me see why the structure is off